cmd/configuration/configuration.go

```diff
@@ -141,13 +138,27 @@ func trim(hostname string) string {
 }
 
 func validateEndpointName(domain string) bool {
-	RegExp := regexp.MustCompile(`^(([a-zA-Z]{1})|([a-zA-Z]{1}[a-zA-Z]{1})|([a-zA-Z]{1}[0-9]{1})|([0-9]{1}[a-zA-Z]{1})|([a-zA-Z0-9][a-zA-Z0-9-_]{1,61}[a-zA-Z0-9]))\.([a-zA-Z]{2,6}|[a-zA-Z0-9-]{2,30}\.[a-zA-Z
- ]{2,3})$`)
-	if RegExp.MatchString(domain) || domain == "localhost" {
+	var validate = false
+	if domain == "localhost" {
 		return true
 	}
```

Comment:
Instead of doing this explicitly , can any of the is package functions validate it. eg : is.DNSName.Validate

Reply:
no it doesn't validate